x/twap/types/keys.go

```diff
@@ -28,6 +28,7 @@ var (
 	mostRecentTWAPsNoSeparator         = "recent_twap"
 	historicalTWAPTimeIndexNoSeparator = "historical_time_index"
 	historicalTWAPPoolIndexNoSeparator = "historical_pool_index"
+	expectedLenOfKeySeparators         = 5
```

Comment:
nit: at first I thought that this refers to the number of characters key separators take
```suggestion
	expectedKeySeparatedParts         = 5
```